immutable/index.ts

```diff
@@ -1,5 +1,4 @@
-// @ts-ignore
-import useSWR from 'swr'
+import useSWR from '../src/use-swr'
```

Comment:
It's necessary to use `import useSWR from 'swr'`, so the `swr` core doesn't get bundled into this sub-package (`swr/immutable`). Preact also uses this approach.

Reply:
Thank you! I understand the intention.